scripts/rambox.sh

```diff
@@ -1,5 +1,9 @@
 # Rambox workspace browser - https://rambox.pro
 sudo apt install gconf2
-wget -O rambox.deb https://github.com/ramboxapp/community-edition/releases/download/0.6.2/Rambox-0.6.2-linux-amd64.deb
+curl --get https://api.github.com/repos/ramboxapp/community-edition/releases/latest | grep "browser_download" | grep "amd64.deb" | tr -d '"'  > cout
+cut -c 29- cout > clink
+rm cout
+wget -O rambox.deb -i clink
+rm clink
```

Comment:
in this way we can ALWAYS get the latest releases?

Reply:
Yes, this call the api of github and trunk the output to get only the browser download. This line:
https://github.com/YetAnotherPackageInstaller/YAPI/blob/5b990308a09cee3765294d751618953401b0d171/scripts/rambox.sh#L3
this command is used with this url:
`https://api.github.com/repos/ramboxapp/community-edition/releases/latest`
the last two strings specify the latest releases of that repository. The output shuold be a json whit a bunch of infos about that and we are interested in the browser download so first grep take the line where the **browser_download** are. This can be return more than one line so I put a second grep in that pipeline to return only the output of **amd64.deb** (this could be even a **amd64.zip** or something like that) and this output is something like this:
```     
"browser_download_url": "https://github.com/ramboxapp/community-edition/releases/download/0.6.4/Rambox-0.6.4-linux-amd64.deb"
```
You can see the `"` that enclosure the link so it must be delete this character with the command **tr** and at last this output will be printed in a file called **cout**.
Than I cut the useless character and take only the url and print that url into a file called clink.

Reply:
okey